llm_toolkit/prompt_builder.py

```diff
@@ -292,6 +297,11 @@ def _format_fixer_problem(self, raw_code: str, error_desc: Optional[str],
     # We are adding errors one by one until we reach the maximum prompt size
     selected_errors = []
     for error in errors:
+      # Skip C only errors.
+      if (FALSE_EXTERN_KEYWORD_ERROR in error or
+          FALSE_FUZZED_DATA_PROVIDER_ERROR in error):
+        continue
```

Comment:
These are the 2 C only errors caused by `#include <fuzzer/FuzzedDataProvider.h>` and `extern "C"`:

```
In file included from {file_path}:{line_num}:
/usr/local/lib/clang/15.0.0/include/fuzzer/FuzzedDataProvider.h:16:10: fatal error: 'algorithm' file not found
#include <algorithm>
         ^~~~~~~~~~~
```

```
{file_path}:{line_num}:{char_pos}: error: expected identifier or '('
extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
       ^
```

Reply:
I see.
IIRC, this error is not uncommon and require `jcc` fixes.
Let's leave a TODO here so that we can address this in the future. E.g.,
```python
# TODO(Dongge): Fix JCC to address this.
# https://github.com/google/oss-fuzz-gen/pull/208/files/a0c0db2fd5860e6e4d434467c5ec9f949ee2cff1#r1571651507
```

Reply:
Noted.